howto/anbox/develop-addon.md

```diff
@@ -0,0 +1,41 @@
+Developing and testing addons using AMS may take a lot of time. Instead, a container with `--devmode` enabled can provide a safe environment to test addons and their scripts without having to upload the addon to the AMS.
+
+This guide explains how to use a container in development mode to develop and test an addon using Anbox runtime. See [development mode](https://discourse.ubuntu.com/t/17763#dev-mode) to learn more about development mode enabled containers.
+
+## Launch a container with `--devmode` enabled
+
+Start a raw container with `--devmode` enabled:
```

Comment:
That's pretty redundant with the title. Or the title is redundant with the first sentence, depending on how you see it.